### ISSUE TITLE
Range slider broken in 1.5.0.3

### ISSUE THREAD

**Issue by 101010f** [CalderaWP/Caldera-Forms#1313]
The range slider seems to be stuck at the initial value and works only after I restore down/maximize browser window for some reason.

**Comment by Shelob9** [comment]
@101010f What browser are you testing in? 

Is the range slider on multi-page form's 2nd or higher page? If so then this is a dupe of  #1230, else this is a new issue.

Also, as a developer I hate that field type. As a user, I love it :)

**Comment by 101010f** [comment]
Chrome and yes the sliders are on a second page. I mainly use sliders as a replacement for number selectors where needed.

**Comment by Shelob9** [comment]
OK, so closing this as a dupe of #1230 then, which I just fixed :) Your clue about zoom in and zoom out was actually really helfpul.

**Comment by 101010f** [comment]
Great. :)
